### ISSUE TITLE
API docs

### ISSUE THREAD

**Issue by MauroDiamantino** [ljean/modbus-tk#187]
Hello, have you published docs where the API (each class and method) is explained? If not, should the use of the library be learn through the examples?

**Comment by ljean** [comment]
Yes the docs should be improved but we have some examples in the example folder for getting started.

**Comment by jacksonmatheson** [comment]
I've started working on this on a forked repo. Will update when I'm getting closer to something that resembles a wiki.
#191
